lnwallet/chainfee/estimator.go

```diff
@@ -430,7 +472,20 @@ func (b *BitcoindEstimator) EstimateFeePerKW(
 //
 // NOTE: This method is part of the Estimator interface.
 func (b *BitcoindEstimator) RelayFeePerKW() SatPerKWeight {
-	return b.minFeeManager.fetchMinFee()
+	// Compare the minimum relay fee against the filterManager's suggested
+	// feerate and return the higher of the two.
+	minFee := b.minFeeManager.fetchMinFee()
+
+	filterFee, err := b.filterManager.fetchMedianFilter()
```

Comment:
Duplicated with the `btcd` version. Can roll this into a method within `filterManager`, then pass in the `minFee` for the comparison.

Reply:
Fixed